daemon/cmd/kube_proxy_replacement.go

```diff
@@ -391,15 +403,16 @@ func probeCgroupSupportTCP(strict, ipv4 bool) {
 		}
 
 		if strict {
-			scopedLog.Fatal(msg)
+			return fmt.Errorf(msg)
```

Comment:
nit, we probably don't need the `scopedLog` var above any more, can combine into the Warn below.